website/docs/en/guide/basic/configure-rslib.mdx

```diff
@@ -1 +1,209 @@
+
+## Configuration Structure
+
+The configuration structure of Rslib looks like this:
+
+```js title="rslib.config.mjs"
+export default {
+  // Array of library configurations.
+  // Each object represents a set of independent configurations for each output format.
+  // Different Rslib and Rsbuild configurations can be specified in each object.
+  lib: [
+    // The first set of configurations
+    {
+      // Independent Rslib configuration of ESM format
+      format: 'esm', // options for the output format of JavaScript files
+      bundle: true, // options for the build type
+      autoExtension: true, // options for the extensions of JavaScript files
+      syntax: 'esnext', // options for the target syntax
+      externalHelpers: false, // options for externalizing helper functions
+      umdName: 'RslibUmdExample', // options for the export name of the UMD library
+      autoExternal: {
+        // options for automatically externalizing third-party dependencies
+      },
+      redirect: {
+        // options for redirecting import paths
+      },
+      dts: {
+        // options for generating TypeScript declaration files
+      },
+      shims: {
+        // options for compatibility shims
+      },
+      banner: {
+        // options for the file banner
+      },
+      footer: {
+        // options for the file footer
+      },
+      // Independent Rsbuild configuration
+      output: {
+        // options for build outputs
+      },
+      source: {
+        // options for source code parsing and compilation
+      },
+      tools: {
+        // options for the low-level tools
+      },
+      plugins: [
+        // configure Rsbuild plugins
+      ],
+      // other independent Rsbuild configuration options
+    },
+    {
+      // The second set of configurations of CJS format
+      format: 'cjs',
+      // ... other independent Rslib / Rsbuild configurations
+    },
+    // ... Extra sets of configurations (if needed)
+  ],
+  // Shared Rsbuild configuration which will be merged into each of the above lib configuration objects
+  output: {
+    // options for build outputs
+  },
+  source: {
+    // options for source code parsing and compilation
+  },
+  tools: {
+    // options for the low-level tools
+  },
+  plugins: [
+    // configure Rsbuild plugins
+  ],
+  // ... other Rsbuild configuration options
+};
+```
+
+The configuration file exports a default object that includes a `lib` array and shared Rsbuild configuration options. You can find detailed descriptions of all configs on the [Configure Overview](/config/) page.
+
+- **Lib Configuration (required)**: The `lib` array contains multiple objects, each representing a set of independent configurations tailored for different output formats, such as ESM and CJS. Each object within the `lib` array can specify distinct configurations for both Rslib and Rsbuild.
```

Comment:
Mentioning both Rslib and Rsbuild here might be a bit confused for users. Rslib's config extends Rsbuild's, so we could tell user you can specify distict Rslib config here. And then tell them Rslib is a superset of Rsbuild in `lib`.